tests/common/fixtures/elasticsearch.py

```diff
@@ -38,16 +32,11 @@ def es_connect():
 
 @pytest.fixture(scope="session", autouse=True)
 def init_elasticsearch(request):
-    """Initialize the test (old) Elasticsearch index once per test session."""
-    client = _es_client()
     """Connect to the newer v6.x instance of Elasticsearch once per test session"""
```

Comment:
This comment could use updating since ES6 is now the only cluster rather than the "newer" one.